ansible/roles-infra/infra-osp-resources-destroy/tasks/heat.yml

```diff
@@ -7,21 +7,18 @@
     OS_PROJECT_DOMAIN_ID: "{{ osp_auth_project_domain }}"
     OS_USER_DOMAIN_NAME: "{{ osp_auth_user_domain }}"
   block:
-        - name: Check if Heat stack already exists
-          command: >-
-            openstack stack show -f json base-stack-{{ guid }}
-          register: r_show
-          failed_when: false
-          changed_when: false
-          tags: heat
+  - name: Get a list of stacks in the project
+    command: >-
+      openstack stack list --property tenant={{ osp_project_info[0].id }} -f value --column ID
+    register: r_stacks
```

Comment:
can we add a comment about why `command` is necessary here VS built-in ansible module?

Reply:
Comments added.